hs-bindgen/tests/Misc.hs

```diff
@@ -36,6 +39,11 @@ goldenVsStringDiff_ name fp action = goldenTestSteps name correct action cmp upd
         | xss == yss = return Nothing
         | otherwise  = return $ Just $ ansidiff xss yss
 
+    normalise :: String -> String
+    normalise [] = []
+    normalise ('\r':'\n':r) = '\n' : normalise r
```

Comment:
The question is still, *what* generates `\r\n` endings, even on Windows. We *should not* generate  them in the first place. That will  be a problem for users of  the tool too.